shared/chat/conversation/messages/wrapper/container.js

```diff
@@ -75,13 +75,19 @@ const mergeProps = (stateProps, dispatchProps) => {
     (previous.type === 'text' || previous.type === 'deleted')
 
   const oldEnough =
-    previous &&
-    previous.timestamp &&
-    message.timestamp &&
+    !!previous &&
```

Comment:
why add these !!?

Reply:
Flow gives a lot of errors about `All branches are incompatible` with `boolean` if I remove any of these

Reply:
k, likely fixable by doing const oldEnough = !!(...)

Reply:
done!